### PR TITLE
Websockets docs example

### PR DESCRIPTION
**Pull request by lucasmreis**
Some months ago I tried to use WebSockets with Suave and had a hard time finding any documentation about it ( http://lucasmreis.github.io/blog/does-elm-harmonize-with-f/#a-server  ).

I'm trying here to show how write a very simple and easy websockets server. Do you feel it's enough? Is it well explained? Does it need more details, or cover more cases?

I also updated the example in the examples folder to match the example in the documentation.

Thanks!

### CODE REVIEW

File: docs/websockets.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Realtime Messages With WebSockets
+---------------------------------
+
+It's easy to set up WebSockets with Suave.
+
+First, define a function that takes `WebSocket` and `HttpContext` typed parameters, and returns a socket computation expression:
+
+{% highlight fsharp %}
+let ws (webSocket : WebSocket) (context: HttpContext) =
```
Comment (haf) [inline docs/websockets.md]: We should have the `open`-statements in the same, too.

**Comment by haf** [comment]
@lucasmreis This is great! Thank you. I think that if we add an open-statement, we're good to go.

File: docs/websockets.md (hunk 1 above)
Comment (lucasmreis) [inline docs/websockets.md]: What do you mean by the `open`-statements? An introduction paragraph? (sorry, non-native english speaker here :) )

Comment (haf) [inline docs/websockets.md]: Just to open Suave.WebSockets so the code compiles

**Comment by ademar** [comment]
Also can we format the code with 2 spaces and perhaps explain a bit what does the example does. Thanks!

**Comment by lucasmreis** [comment]
Ok! I'll do it this night. Would you prefer the explanations of the example through comments in the example itself, or should I bring it to the documentation page?

**Comment by haf** [comment]
I tend to like comments in the code unless the web site has a good way to annotate lines of code, which ours doesn't have right now.

**Comment by haf** [comment]
Thank you Lucas!

**Comment by lucasmreis** [comment]
Thank you too! When will the website be deployed?

**Comment by lucasmreis** [comment]
@haf the website is still in the older version :)

**Comment by haf** [comment]
Deployed

**Pull request merged**